src/tests/test_parameterlist_oceandefault.C

```diff
@@ -0,0 +1,87 @@
+#include "TestDefinitions.H"
+
+//------------------------------------------------------------------
+namespace // local unnamed namespace (similar to static in C)
+{
+    RCP<Ocean> ocean;
+    RCP<Epetra_Comm>  comm;
+}
+
+//------------------------------------------------------------------
+TEST(ParameterListOcean, DefaultInitialization)
```

Comment:
Can this be moved to the other file? THCM may complain (being a singleton), but it should still work as long as you don't try to use the old instance, right? Same for the THCM test files.

Reply:
I would prefer that, but when I initially had them in the same file, the Fortran code segfaulted and prints an error about variables already being initialised. If you know how I can make it work, I'm all ears.